setup.py

```diff
@@ -462,6 +462,21 @@ def parse_setuppy_commands():
                   ' '.join(sys.argv[1:])))
     return True
 
+def check_setuppy_command():
+    run_build = parse_setuppy_commands()
+    if run_build:
+        try:
+            import numpy
+            import cython
```

Comment:
We don't use `import cython` anywhere, and I don't think it necessarily has to succeed. Cython can be installed, e.g., in the conda `base` environment, while one is trying to build in a fresh env with only `numpy` and `pybind11` installed. We need Cython as a command-line tool, not a Python package AFAIK.

If it's missing completely then a sane error will already be raised - see `tools/cythonize.py`.